html/semantics/scripting-1/the-script-element/microtasks/checkpoint-after-shadowrealmglobalscope-onerror.any.js

```diff
@@ -0,0 +1,32 @@
+// META: title=Microtask checkpoint after ShadowRealm onerror events
+// META: global=shadowrealm
+
+// Adapted from first part of ./resources/checkpoint-after-error-event.js.
```

Comment:
I have not verified that the expected behavior is correct, but I assume it is if it matches the existing test